web/app/scripts/filterbar/numeric-range-directive.js

```diff
@@ -15,17 +15,18 @@
             link: function(scope, elem, attrs, ctlArray) {
                 init();
 
+                // restore previously set filter selection on page reload
+                scope.$on('driver.filterbar:restored', function(event, filter) {
+                    if (filter.label === scope.label) {
+                        scope.filter = filter.value;
+                        scope.isMinMaxValid();
+                    }
+                });
+
                 function init() {
                     scope.filter = {};
                     scope.error = {};
-
-                    // restore previously set filter selection on page reload
-                    scope.$on('driver.filterbar:restored', function(event, filter) {
-                        if (filter.label === scope.label) {
-                            scope.filter = filter.value;
-                            scope.isMinMaxValid();
-                        }
-                    });
+                    scope.filter = FilterState.get(scope.label);
```

Comment:
I don't see the reason for the changes to this file. `FilterState` should not be referenced in the specific filter directives; that's managed at a higher level. The numeric range filters were working fine.